anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -106,6 +107,43 @@ def get_infographics_data_from_cache_by_road_segment(road_segment_id, years_ago)
         return {}
 
 
+def get_cache_retrieval_query(params: RequestParams):
+    res = params.resolution
+    if res == BE_CONST.ResolutionCategories.SUBURBAN_ROAD:
+        return (
+            db.session.query(InfographicsRoadSegmentsDataCache)
+            .filter(
+                InfographicsRoadSegmentsDataCache.road_segment_id
+                == int(params.location_info["road_segment_id"])
+            )
+            .filter(InfographicsRoadSegmentsDataCache.years_ago == int(params.years_ago))
+        )
+    else:
+        msg = f"Cache unsupported resolution: {res}, params:{params}"
+        logging.error(msg)
+        raise ValueError(msg)
+
+
+def get_infographics_data_from_cache_by_location(request_params: RequestParams) -> Dict:
+    logging.error("get_infographics_data_from_cache_by_location: not yet implemented")
```

Comment:
If it's not yet implemented, why to include the code?

Reply:
Correct. This log is not relevant any more. Removed.